Common/src/Common/assets/js/inline/type-of-licence.js

```diff
@@ -58,26 +58,32 @@ OLCS.ready(function () {
                     return operatorType.filter(':checked').val() === 'lcat_psv';
                 },
                 'selector:#ltyp_sr_radio_group': function () {
-                    return operatorType.filter(':checked').val() === 'lcat_psv';
+                    var shouldShow = operatorType.filter(':checked').val() === 'lcat_psv';
+                    if (shouldShow) {
+                        $('#ltyp_sr_radio_group').removeAttr('style');
+                    } else {
+                        $('#ltyp_sr_radio_group').css('display', 'none');
+                    }
+                    return shouldShow;
```

Comment:
How was the visibility handled before?

Reply:
jQuery toggled the element between `style=”display: none;"` and  `style=”display: block;"`

`style=”display: block;"` unhides the element but breaks the alignment.

Reply:
Ah okay, would this be limited to just this radio?

Is there a solution where we can fix it for all the radios that are affected? It's because GDS have changed to `display: flex` I'd imagine.

Reply:
No other instances have been reported, ideally i would need another example to be able to create some code what would be generic and reliably fix it in other places in the app.